sklearn/metrics/tests/test_ranking.py

```diff
@@ -13,13 +13,11 @@
 from sklearn.utils.validation import check_array, check_consistent_length
 from sklearn.utils.validation import check_random_state
 
-from sklearn.utils.testing import assert_raises, clean_warning_registry
-from sklearn.utils.testing import assert_raise_message
-from sklearn.utils.testing import assert_equal
-from sklearn.utils.testing import assert_almost_equal
-from sklearn.utils.testing import assert_array_equal
-from sklearn.utils.testing import assert_array_almost_equal
-from sklearn.utils.testing import assert_warns
+from sklearn.utils.testing import (assert_raises, clean_warning_registry,
+                                   assert_raise_message, assert_equal,
+                                   assert_almost_equal, assert_array_equal,
+                                   assert_array_almost_equal, assert_warns,
+                                   assert_allclose)
```

Comment:
the code was using assert_array_almost_equal to compare scalars... I can revert it, but let me know what you think about the pr description (just edited)